rubicon_ml/client/rubicon.py

```diff
@@ -73,9 +100,9 @@ def _get_github_url(self):
     def _create_project_domain(
         self,
         name: str,
-        description: str,
-        github_url: str,
-        training_metadata: Union[List[Tuple], Tuple],
+        description: Optional[str],
+        github_url: Optional[str],
+        training_metadata: Optional[Union[List[Tuple], Tuple]],
     ):
         """Instantiates and returns a project domain object."""
         if self.config.is_auto_git_enabled and github_url is None:
```

Comment:
this can be `self.is_auto_git_enabled` now, right?

Reply:
Related to issue #387 - that method is in the `Base` class not here. Let me look.